content/chainguard/chainguard-images/security-advisories/index.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "How to Use Chainguard Security Advisories"
+linktitle: "Security Advisories"
+type: "article"
+description: "Article outlining how one can explore and use the Security Advisories found on the Chainguard Image Directory."
+date: 2023-12-27T11:07:52+02:00
+lastmod: 2023-12-27T11:07:52+02:00
+draft: false
+tags: ["Overview", "Product", "Chainguard Images", "CVE"]
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 740
+toc: true
+---
+
+Say you use an image scanning tool like Grype or Docker Scout to determine what, if any, vulnerabilities are present within a given Chainguard Image. Chainguard Images typically report few or zero CVEs, but some Images — typically our Production Images, which consist of older versions of a given software and are therefore prone to vulnerabilities — may report more.
+
+It may not be immediately clear to users whether a CVE found within a given Chainguard Image is likely to cause security issues in their software. After all, scan results can include [false positives](/chainguard/chainguard-images/scanners/false-results/), or true positives may be removed from later versions of the Image. To help demystify the nature of CVEs within Chainguard Images, we've created a self-service [Security Advisories page](https://images.chainguard.dev/security) that lists every security advisory published for Chainguard Images. Having this information available allows you to view whether Chainguard is aware of a specific vulnerability reported to exist within a Chainguard Image and whether we've mitigated or are planning to mitigate the CVE.
```

Comment:
I think we can frontload more that scanners may misreport vulnerabilities in Chainguard Images, and that you should check the Security Advisories page for any that you want to learn more about, as many of them have been fixed.